src/pathfinding_service/model/token_network.py

```diff
@@ -260,12 +251,15 @@ def handle_channel_balance_update_message(
         )
 
     def handle_channel_fee_update(self, message: FeeUpdate) -> None:
+        channel_id = message.canonical_identifier.channel_identifier
+        participants = self.channel_id_to_addresses[channel_id]
+        other_participant = (set(participants) - {message.updating_participant}).pop()
```

Comment:
Would it make sense to move this into a function? I think we do this a lot in the `service` as well

Reply:
I can't see such a case in `pathfinding_service/service.py`. The `UpdatePFS` already includes the other participant, so that logic is not necessary, there.